src/app/ui/components/containers/footers/footer.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import type { Meta } from '@storybook/react';
+import { Flex, styled } from 'leather-styles/jsx';
+
+import { LeatherButton } from '@app/ui/components/button';
+import { AvailableBalance } from '@app/ui/components/containers/footers/available-balance';
+
+// import { customViewports } from '../../../../../../.storybook/viewports';
+import { Footer as Component } from './footer';
+
+const meta: Meta<typeof Component> = {
+  component: Component,
+  tags: ['autodocs'],
+  title: 'Design System/Containers/Footer',
+  parameters: {
```

Comment:
I was trying to get rid of this 'Args table with interactive controls...` message here but its not working:

![Screenshot 2024-01-19 at 06 21 06](https://github.com/leather-wallet/extension/assets/2938440/ee22975e-4894-40c3-a256-d9c68ef02822)

I tried to restrict the width of the viewport for these stories also by setting a custom `viewport` but didn't get it working either. Improvements for the future maybe